whitepaper/Cosmology/wl.tex

```diff
@@ -10,152 +10,425 @@
+Observing strategies have been developed for suppressing these systematics in
+current lensing surveys, such as the Deep Lens
+Survey\footnote{\url{dls.physics.ucdavis.edu}}.  These and new methods will be
+applied to the LSST survey.
 
-\subsection{Target Selection}
+To leading order, we can express the effect of shear systematics in the observed
+shear $\gamma^\mathrm{obs}_i$ as a small linear perturbation of the true shear
+$\gamma_i$,
 
-Image quality must be uniformly good in the bands used for weak lens shear.  These will be 
-the $r$ and $i$ bands.   Depending on the current weather and seeing, the scheduler
-will have a list of priorities for next-field, based on prior history of coverage.
-Nearby fields in need of coverage in these bands will be given high priority if the
-seeing is better than some specified value, likely 0.7 arcsec FWHM.
+$$ \gamma_i^\mathrm{obs} = (1+m_i) \gamma_i + c_i, $$
 
+where $m_i$ is the multiplicative and $c_i$ is the additive systematic in the
+ith shear component.  From the LSST SRD requirements on residual systematics in
+the shear-shear correlation function one can specify the level of residual shear
+systematics at which statistical uncertainties become subdominant.  Over the
+sample of 3-4 billion galaxies, the shear systematics must be below 3 parts in
+10,000 for additive shear $|c|$, and 3 parts in 1000 for multiplicative shear
+$|m|$.  Each visit to a sky patch encounters these systematics.  In particular,
+each re-visit to a given field generates the same CCD-based additive shear
+systematic.  Some observing strategies can effectively randomize these over all
+visits to a field.  It is important to note that the full survey shear-shear
+correlation error due to these systematics is expected to be no better than the
+corresponding systematic in any given field after all re-visits to that field.
+This is because the useful angular scales in cosmic shear are less than a field
+radius, and the systematics floor in shear-shear correlation is set therefore by
+the floor in any one typical field.  Below we discuss the observing strategies
+for suppressing shear systematics and metrics for their success.
+
+
+\subsection{Target Selection}
+
+Image quality must be uniformly good in the bands used for weak lensing shear.
+These will be mainly the $r$ and $i$ bands, though it is possible that the $z$
+band will also be used for shear measurement.  The decision on which field to
+observe next must be based mainly on its weak lensing priority \citep[Sec 3.1
+and Figure 14.4]{2009arXiv0912.0201L}.  Depending on the current weather and
+seeing, the scheduler will have a list of priorities for next-field, based on
+prior history of coverage.  The relevant parameters are seeing, depth, and
+camera rotation angle with respect to North and to zenith. Nearby fields in need
+of coverage in these bands should be given high priority if the seeing is better
+than some specified value, likely 0.7 arcsec FWHM \citep[Sec
```

Comment:
Yes, image quality in the next-field decision is currently part of the 
scheduler algorithm.  So airmass plays a role.



On 6/29/16 6:50 PM, Rachel Mandelbaum wrote:
> In whitepaper/Cosmology/wl.tex
> <https://github.com/LSSTScienceCollaborations/ObservingStrategy/pull/478#discussion_r69058335>:
>
>> +the floor in any one typical field.  Below we discuss the observing strategies
>> +for suppressing shear systematics and metrics for their success.
>> +
>> +
>> +\subsection{Target Selection}
>> +
>> +Image quality must be uniformly good in the bands used for weak lensing shear.
>> +These will be mainly the $r$ and $i$ bands, though it is possible that the $z$
>> +band will also be used for shear measurement.  The decision on which field to
>> +observe next must be based mainly on its weak lensing priority \citep[Sec 3.1
>> +and Figure 14.4]{2009arXiv0912.0201L}.  Depending on the current weather and
>> +seeing, the scheduler will have a list of priorities for next-field, based on
>> +prior history of coverage.  The relevant parameters are seeing, depth, and
>> +camera rotation angle with respect to North and to zenith. Nearby fields in need
>> +of coverage in these bands should be given high priority if the seeing is better
>> +than some specified value, likely 0.7 arcsec FWHM \citep[Sec
>
> How does airmass play into this? If some field is high priority as
> next-field based on the parameters you mentioned, but would have to be
> observed at high airmass, is that taken into account at all?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/LSSTScienceCollaborations/ObservingStrategy/pull/478/files/68564bd57311f22658256026e5e6e6a39c2554f4#r69058335>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACA6YlRkgQ6zM6fjfMiJ5Sjn_0BdWL7uks5qQyDMgaJpZM4I9hd->.
>